scripts/feature_tests/node_reuse/Makefile

```diff
@@ -0,0 +1,10 @@
+export NUM_OF_MASTER_REPLICAS := 3
+export NUM_OF_WORKER_REPLICAS := 1
+
+all: node_reuse_kcp node_reuse_md
+
+node_reuse_kcp:
+	./node_reuse_kcp.sh
+
+node_reuse_md:
+	./node_reuse_md.sh
```

Comment:
Can't we have a single target script but export the `ACTION` here before calling the script? That would avoid duplicating the same script.

Reply:
Good point, thanks, fixed!